viper.go

```diff
@@ -1162,34 +1168,38 @@ func (v *Viper) InConfig(key string) bool {
 }
 
 // SetDefault sets the default value for this key.
-// SetDefault is case-insensitive for a key.
+// SetDefault's case-sensitivity for key is determined by viper.keyCaseSensitivity.
```

Comment:
I would retain the original comment and add something like this:
`This behavior can be modified by viper.SetKeyCaseSensitivity`